tfx/utils/path_utils.py

```diff
@@ -80,3 +81,8 @@ def serving_model_path(output_uri: Text) -> Text:
   else:
     # If dir doesn't match estimator structure, use serving model root directly.
     return serving_model_dir(output_uri)
+
+def log_output_dir(output_uri: Text) -> Text:
```

Comment:
for generic trainer, we just pass the working dir to user, what structure is up to user,

don't need to change path_utils.py